tests/10-issue.js

```diff
@@ -26,95 +26,325 @@ describe('BitstringStatusList Credentials (Issue "statusPurpose: revocation")',
+                });
+              } else {
+                it('If statusSize is not present as a property ' +
+                  'of the credentialStatus, then statusSize MUST ' +
+                  'be processed as 1.',
+                async function() {
+                  this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=If%20statusSize%20is%20not%20present%20as%20a%20property%20of%20the%20credentialStatus%2C%20then%20statusSize%20MUST%20be%20processed%20as%201.';
+                }
+                );
+              }
+            });
+          it('An implementer MAY include the statusReference property.',
+            async function() {
+              this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=An%20implementer%20MAY%20include%20the%20statusReference%20property.';
+              if(!!issuedVc.credentialStatus.statusReference) {
+                it('If present, its value MUST be a URL or an array of URLs ' +
+                    '[URL] which dereference(s) to material related to the ' +
+                    'status.',
+                async function() {
+                  this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=If%20present%2C%20its%20value%20MUST%20be%20a%20URL%20or%20an%20array%20of%20URLs%20%5BURL%5D%20which%20dereference%20to%20material%20related%20to%20the%20status';
+                  const statusReferenceType = typeof (
+                    issuedVc.credentialStatus.statusReference);
+                  statusReferenceType.should.be.oneOf(['string', 'array'],
+                    'Expected statusReference to be an string or an array.');
+                  // TODO test for URLS
+                });
+              } else {
+                this.skip();
+              }
+            });
         });
-        it('MUST be 16kb in size when "encodedList" is decoded',
+        describe('BitstringStatusList Credential', function() {
+          let statusListCredential;
+          before(async function() {
+            statusListCredential = (await getSlc({issuedVc})).slc;
+          });
+          it('When a status list verifiable credential is published, ' +
+            'it MUST be a conforming document, ' +
+            'as defined in [VC-DATA-MODEL-2.0].',
+          async function() {
+            this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=When%20a%20status%20list%20verifiable%20credential%20is%20published%2C%20it%20MUST%20be%20a%20conforming%20document%2C%20as%20defined%20in%20%5BVC%2DDATA%2DMODEL%2D2.0%5D';
+            testSlCredential({slCredential: statusListCredential},
+              'Expected status credential to conform to VCDM 2.0.');
+          });
+          it('The verifiable credential that contains the status ' +
+            'list MAY express an id property that matches the value ' +
+            'specified in statusListCredential for the corresponding ' +
```

Comment:
is there a test here yet?

Reply:
I'm not sure if we should, it doesn't say that it MUST match, I will just skip.